pkg/updater/repository/repository.go

```diff
@@ -241,29 +315,111 @@ func movePackageFromSource(packageName string, rootPath string, sourcePath strin
 	if err != nil {
 		return "", fmt.Errorf("could not move source: %w", err)
 	}
+
+	// Create the associated run path if it doesn't exist
+	if runPath != "" {
+		err = os.MkdirAll(filepath.Join(runPath, packageName), 0766)
```

Comment:
I don't like to have this code twice (and missing the permissions check)

Reply:
It's not the same operation, here we're creating the package version folder in which we'll put the PIDs. Agreed on the permissions, will do